client/task-list/tasks/payments/RemotePayments/index.js

```diff
@@ -120,7 +120,7 @@ export const RemotePayments = ( { query } ) => {
 	};
 
 	const currentMethod = useMemo( () => {
-		if ( ! query.method || isResolving ) {
+		if ( ! query.method || isResolving || ! methods.length ) {
```

Comment:
This change looked unrelated so just checking was it intended?

Reply:
Yeah, this is covering an error this is cropping up currently, although I believe it will be resolved when we're able to tie some of the PRs in flight together.